File: modul-03/python/PRAK305-2310817310003-Mfaridsetiawan.py
hari = 0
jam = 0
menit = 0
detik = 0

variabel = int(input())

if variabel >= 86400:
    hari = variabel // 86400
    variabel %= 86400

if variabel >= 3600:
    jam = variabel // 3600
    variabel %= 3600

if variabel >= 60:
    menit = variabel // 60
    variabel %= 60

detik = variabel

if hari > 0:
    print(f"{hari} hari:{jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: modul-03/bahasa c/PRAK305-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main()
{
    int detik, hari, jam, menit, detik2;

    printf("input\n");
    scanf("%d", &detik);

    hari = detik / (24 * 3600);
    jam = (detik % (24 * 3600)) / 3600;
    menit = (detik % 3600) / 60;
    detik2 = detik % 60;

    if (hari > 0)
    {
        printf("\nOutput \n%d hari %02d:%02d:%02d", hari, jam, menit, detik2);
    }
    else
    {
        printf("\nOutput \n%02d:%02d:%02d", jam, menit, detik2);
    }

    return 0;
}
